### PR TITLE
add revoke-all endpoint

### PR DESCRIPTION
**Pull request by smrutherford**
Adding support for `revoke-all` endpoint: https://docs.nylas.com/v2.0/reference#revoke-all

### CODE REVIEW

File: tests/test_client.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import re
 import json
-from six.moves.urllib.parse import parse_qs
+from six.moves.urllib.parse import parse_qs  # noqa
```
Comment (smrutherford) [inline tests/test_client.py]: This was breaking the Travis CI build. Below is the Travis CI error:

________________________ [pylint] tests/test_client.py _________________________
W:  3, 0: Relative import 'six.moves.urllib.parse', should be 'six.moves' (relative-import)

The error seems to be an open pylint issue. See https://github.com/PyCQA/pylint/issues/2180

File: README.md [hunk 2]
```diff
@@ -83,6 +83,17 @@ use the `revoke_token` method on APIClient
   client.revoke_token()
 ```
 
+**Revoke either all tokens or all but one token**
```
Comment (pfista) [inline README.md]:
```suggestion
**Revoke all tokens or all but one token**
```

File: tests/test_client.py (hunk 1 above)
Comment (singingwolfboy) [reply tests/test_client.py]: I suggested switching from Pylint to Black in #108. That would resolve the problem, as well 😄

File: nylas/client/client.py [hunk 3]
```diff
@@ -142,6 +143,22 @@ def revoke_token(self):
         self.auth_token = None
         self.access_token = None
 
+    def revoke_all_tokens(self, keep_access_token=None):
+        account = self.account
+        client_id = self.app_id
```
[the remark this answers is not in the document]
Comment (smrutherford) [reply nylas/client/client.py]: yep, good point! i've updated it 🙂

File: README.md (hunk 2 above)
Comment (smrutherford) [reply README.md]: done!

File: tests/test_client.py (hunk 1 above)
Comment (smrutherford) [reply tests/test_client.py]: haha good to know! Maybe one day lol

**Pull request merged**